keyboards/handwired/hnah40rgb/info.json

```diff
@@ -0,0 +1,100 @@
+{
+    "keyboard_name": "Hnah40", 
+    "url": "",
+    "maintainer": "HnahKB", 
+    "width": 12, 
+    "height": 4, 
+    "layouts": {
+        "LAYOUT_all": {
+            "key_count": 42,
+            "layout": [
+                {"label":"Esc", "x":0, "y":0}, 
+                {"label":"Q", "x":1, "y":0}, 
+                {"label":"W", "x":2, "y":0}, 
+                {"label":"E", "x":3, "y":0}, 
+                {"label":"R", "x":4, "y":0}, 
+                {"label":"T", "x":5, "y":0}, 
+                {"label":"Y", "x":6, "y":0}, 
+                {"label":"U", "x":7, "y":0}, 
+                {"label":"I", "x":8, "y":0}, 
+                {"label":"O", "x":9, "y":0}, 
+                {"label":"P", "x":10, "y":0}, 
+                {"label":"BSPC", "x":11, "y":0}, 
+                {"label":"Caps Lock", "x":0, "y":1, "w":1.5}, 
+                {"label":"A", "x":1.5, "y":1}, 
+                {"label":"S", "x":2.5, "y":1}, 
+                {"label":"D", "x":3.5, "y":1}, 
+                {"label":"F", "x":4.5, "y":1}, 
+                {"label":"G", "x":5.5, "y":1}, 
+                {"label":"H", "x":6.5, "y":1}, 
+                {"label":"J", "x":7.5, "y":1}, 
+                {"label":"K", "x":8.5, "y":1}, 
+                {"label":"L", "x":9.5, "y":1}, 
+                {"label":"Enter", "x":10.5, "y":1, "w":1.5}, 
+                {"label":"Shift", "x":0, "y":2, "w":1.75}, 
+                {"label":"Z", "x":1.75, "y":2}, 
+                {"label":"X", "x":2.75, "y":2}, 
+                {"label":"C", "x":3.75, "y":2}, 
+                {"label":"V", "x":4.75, "y":2}, 
+                {"label":"B", "x":5.75, "y":2}, 
+                {"label":"N", "x":6.75, "y":2}, 
+                {"label":"M", "x":7.75, "y":2}, 
+                {"label":",", "x":8.75, "y":2}, 
+                {"label":".", "x":9.75, "y":2}, 
+                {"label":"Shift", "x":10.75, "y":2, "w":1.25}, 
+                {"label":"Ctrl", "x":0, "y":3, "w":1.25}, 
+                {"label":"Win", "x":1.25, "y":3, "w":1}, 
+                {"label":"Alt", "x":2.25, "y":3, "w":1.25}, 
+                {"label":"Space", "x":3.5, "y":3, "w":2.25}, 
+                {"label":"Space", "x":5.75, "y":3, "w":2.75},
+                {"label":"App", "x":8.5, "y":3, "w":1.25}, 
+                {"label":"Fn", "x":9.75, "y":3, "w":1}, 
+                {"label":"Ctrl", "x":10.75, "y":3, "w":1.25}
+            ]
+        },
+        "LAYOUT_ansi": {
+            "key_count": 40,
```

Comment:
It's actually 39.

```suggestion
            "key_count": 39,
```

Reply:
thank you, i just coppy layout all and forgot edit key count 😄